smac/main/smbo.py

```diff
@@ -137,6 +138,11 @@ def used_target_function_walltime(self) -> float:
         """Returns how much walltime the target function spend so far."""
         return self._used_target_function_walltime
 
+    @property
+    def used_target_function_cputime(self) -> float:
+        """Returns how much cputime the target function spend so far."""
```

Comment:
describe that CPU time generally means time spent on the hardware and not just CPU (whether its CPU, TPU, GPU or whatever)